augur/tasks/github/releases/core.py

```diff
@@ -23,7 +23,7 @@ def get_release_inf(repo_id, release, tag_only):
 
 
         release_inf = {
-            'release_id': release['id'],
+            'release_id': str(release['id']).strip(),
```

Comment:
I think this helps to ensure that strings are interpreted as strings, and spaces are removed. So, it should eliminate "false duplicates".